class.ext_update.php

```diff
@@ -0,0 +1,71 @@
+<?php
```

Comment:
Is there a reason as to why you did not use an upgrade wizard?
https://docs.typo3.org/typo3cms/CoreApiReference/ApiOverview/UpdateWizards/Index.html
Use of ext_update is discouraged.